src/app/components/animated-tick-icon.tsx

```diff
@@ -0,0 +1,32 @@
+import { styled } from 'leather-styles/jsx';
+
+const defaultSize = '20px';
+
+interface AnimatedTickProps {
+  size?: string;
+}
+export function AnimatedTick({ size = defaultSize }: AnimatedTickProps) {
```

Comment:
Shouldn't this be located in the `ui` folder with other icons?

Reply:
Yep, UI didn't exist when I started